sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -85,21 +162,50 @@
   protected final String fileNamingTemplate;
 
   /**
-   * Construct a FileBasedSink with the given base output filename and extension.
+   * Construct a FileBasedSink with the given base output filename and extension. A
+   * {@link OutputWrapperFactory} of type {@link CompressionType#UNCOMPRESSED} will be used.
    */
   public FileBasedSink(String baseOutputFilename, String extension) {
     this(baseOutputFilename, extension, ShardNameTemplate.INDEX_OF_MAX);
   }
 
   /**
+   * Construct a FileBasedSink with the given base output filename, extension, and
+   * {@link OutputWrapperFactory}.
+   */
+  public FileBasedSink(String baseOutputFilename, String extension,
+      OutputWrapperFactory outputWrapperFactory) {
+    this(baseOutputFilename, extension, ShardNameTemplate.INDEX_OF_MAX, outputWrapperFactory);
+  }
+
+  /**
    * Construct a FileBasedSink with the given base output filename, extension, and file naming
-   * template.
+   * template. A {@link OutputWrapperFactory} of type {@link CompressionType#UNCOMPRESSED}
+   * will be used.
    *
-   * <p>See {@link ShardNameTemplate} for a description of file naming templates.
+   * <p>
+   * See {@link ShardNameTemplate} for a description of file naming templates.
```

Comment:
Gah, hate that, sorry.  Is there an Eclipse formatter publicly available that the beam project uses?  I thought I was using the latest Google formatter, but maybe it's not up to date...

Will fix these manually for now.